build.sbt

```diff
@@ -314,6 +316,15 @@ lazy val protobufV3 = akkaModule("akka-protobuf-v3")
     test in assembly := {}, // assembly runs tests for unknown reason which introduces another cyclic dependency to packageBin via exportedJars
     description := "Akka Protobuf V3 is a shaded version of the protobuf runtime. Original POM: https://github.com/protocolbuffers/protobuf/blob/v3.9.0/java/pom.xml")
 
+lazy val pki =
+  akkaModule("akka-pki")
+    .dependsOn(actor) // this dependency only exists for "@ApiMayChange"
+    .settings(Dependencies.pki)
+    .settings(AutomaticModuleName.settings("akka.pki"))
+    .settings(
+      // The akka-pki artifact was added in Akka 2.6.2, no MiMa checks yet.
+      mimaPreviousArtifacts := Set.empty)
```

Comment:
Shouldn't we set it up correctly right away in Mima.scala? If we want to ignore binary compatibility for now, let's use `disablePlugin(MimaPlugin)` instead for clarity.